packages/eslint-config-react-app/index.js

```diff
@@ -26,13 +26,21 @@ module.exports = {
   plugins: ['import', 'flowtype', 'jsx-a11y', 'react'],
 
   env: {
-    browser: true,
     commonjs: true,
     es6: true,
     jest: true,
     node: true,
   },
 
+  globals: {
+    document: true,
```

Comment:
We need to draw the line somewhere, and I think it's honestly not hard to find usages of them if you intend to convert your app to SSR (which CRA doesn't support out of the box anyway).

It's much harder without a whitelist (since then you don't even know what you're looking for).